src/internal.c

```diff
@@ -5330,8 +5330,10 @@ static INLINE int GrowOutputBuffer(WOLFSSL* ssl, int size)
 #if WOLFSSL_GENERAL_ALIGNMENT > 0
     byte  hdrSz = ssl->options.dtls ? DTLS_RECORD_HEADER_SZ :
                                       RECORD_HEADER_SZ;
-#endif
+    byte align = WOLFSSL_GENERAL_ALIGNMENT;
+#else
     const byte align = WOLFSSL_GENERAL_ALIGNMENT;
```

Comment:
The scan only found a dead code scenario where WOLFSSL_GENERAL_ALIGNMENT is zero. This was to "cleanup" the function for the case when no align is defined (0). I agree that WOLFSSL_GENERAL_ALIGNMENT should never be a negative value. We could add an #if compile - time check for that in a later commit.